include/class-bulk-move-posts.php

```diff
@@ -446,6 +446,215 @@ public static function change_timeout() {
             ini_set( 'max_execution_time', $max_execution_time );
         }
     }
+
+	/**
+	 * Loads the custom Taxonomy by Post Type.
+	 *
+	 * @since 1.3.0
+	 */
+	public static function load_custom_taxonomy_by_post_type() {
+		$bulk_move = BULK_MOVE();
+		check_ajax_referer( $bulk_move::BOX_CUSTOM_TERMS_NONCE, 'security' );
+		$post_type = isset( $_POST['post_type'] ) ? sanitize_text_field( $_POST['post_type'] ) : 'post';
+
+		$taxonomies = get_object_taxonomies( $post_type );
+
+		$args = array(
+			'taxonomy'   => $taxonomies,
+			'hide_empty' => false,
+			'orderby'    => 'name',
+		);
+
+		ob_start();
+        ?>
+		<option class="level-0" value="select">Select Taxonomy&nbsp;&nbsp;</option>
```

Comment:
Make this text available for translation

Reply:
Perfect catch. I made the string translatable and also made the other strings translatable.